src/scenes/PlantsDashboardRouter/components/ZoneLevelDataMap.tsx

```diff
@@ -232,7 +232,7 @@ export default function ZoneLevelDataMap({ plantingSiteId }: ZoneLevelDataMapPro
     }
 
     const baseMap = MapService.getMapDataFromPlantingSite(plantingSite);
-    if (!observation) {
+    if (!observation || !plantingSiteHistory) {
       return baseMap;
```

Comment:
@sgrimm this change is making the recency layer to not work. If there is no plantingSiteHistory we still need to "calculate" the lastObservation for each zone and subzone to show in the map. Sorry I missed this before, I will try to make a fix in a new PR